hedera-node/hedera-app/src/xtest/java/contract/ClassicViewsXTestConstants.java

```diff
@@ -57,12 +67,12 @@ public class ClassicViewsXTestConstants {
     // the token is fungible
     static final Bytes TOKEN_TYPE_FUNGIBLE =
             Bytes.fromHex(SUCCESS_RESPONSE_CODE + "0000000000000000000000000000000000000000000000000000000000000000");
```

Comment:
For all of these `"00....000"` and `"00....001"` why not have named constants `SUCCESS_RETURNING_FALSE` and `SUCCESS_RETURNING_TRUE` so these hex strings only appear once each and after that their uses are self-explanatory?